partiql-lang/src/main/pig/partiql.ion

```diff
@@ -451,17 +459,20 @@ may then be further optimized by selecting better implementations of each operat
 
         (product table_def parts::(* table_def_part 1))
 
+        (product table_name_prefix steps::(* identifier 0))
```

Comment:
Need a better name for this.  "table" adds too much detail, because this thing should be useful for other schema/catalog inhabitants, such as indexes. Just `name_prefix`?  I feel something with "location" in it could be good, but reluctant to suggest `location` by itself, as that could become useful for something else.

Reply:
Agree with better name. Probably not `name_prefix`, `name_qualifier` or just `qualifier` perhaps?